src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class PENamedTypeSymbol : NamedTypeSymbol
         /// </summary>
         private TypeKind _lazyKind;
 
+        /// <summary>
+        /// Lazily initiated HasEmbeddedAttribute
+        /// </summary>
+        private ThreeState _lazyHasEmbeddedAttribute = ThreeState.Unknown;
```

Comment:
My understanding is that uncommon properties are a singleton, and putting it there would mean throughout the API that all PENamedTypeSymbols have this attribute.
If you don't see otherwise, I'll keep the current design.

---
In reply to: [113556704](https://github.com/dotnet/roslyn/pull/18715#discussion_r113556704) [](ancestors = 113556704,113542264)